frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailsTable.tsx

```diff
@@ -17,34 +18,58 @@ import {
   GridContainer,
   GridCell,
 } from "./ObjectDetail.styled";
+import ExternalLink from "metabase/core/components/ExternalLink";
 
 export interface DetailsTableCellProps {
   column: any;
   value: any;
+  row: any;
```

Comment:
The proper type here would be [`Row` (link)](https://github.com/metabase/metabase/blob/f5d85e8131903976d4b4194f7f5de728b522fe9c/frontend/src/metabase-types/types/Dataset.ts#L39)